src/content/posts/wifi-deauth-attack.md/index.md

```diff
@@ -13,35 +13,40 @@ draft: false
 **Initial State:** A client is connected to a ***password securised Wifi***
 
 The goal is to get the wifi password.  
-To acheive this we must:  
-1) Detect WiFi
+To achieve this we must:  
+1) Detect target WiFi signal
 2) Detect connected computer to this WiFi
-3) Send a huge quantity of package to this computer to make it disconnect from the WiFi
-4) Get the password hash when it tries to reconnect to WiFi
-5) bruteforce it :)
+3) Send de-authentication packets to this computer to make it disconnect from the WiFi
+4) Get the password "hash" when it tries to reconnect to WiFi

```

Comment:
5) Get the MIC (Message Integrity Code) wich is a **hash** of several knownable informations included in the network capture and **password**
See the **4 Way Handshake** article for more informations